imsv-docs-astro/src/content/docs/guides/funding-protocols/universal-evm-smart-contract.mdoc

```diff
@@ -2,6 +2,8 @@
 title: Universal EVM Smart Contract
 description: Oveview of Immersve's Universal EVM funding smart contract.
 slug: guides/universal-evm-smart-contract
+funding-protocol:
+  source: https://github.com/immersve/funding-contract-universal-evm
```

Comment:
Question: is this supposed to be a public repo?

Reply:
good question, I've noticed is a private repo as well @ncjones